src/Faker/Provider/es_ES/Person.php

```diff
@@ -4,6 +4,8 @@
 
 class Person extends \Faker\Provider\Person
 {
+    private static $letters=array('T', 'R', 'W', 'A', 'G', 'M', 'Y', 'F', 'P', 'D', 'X', 'B', 'N', 'J', 'Z', 'S', 'Q', 'V', 'H', 'L', 'C', 'K', 'E', 'T');
```

Comment:
`letters` is a too specific name. Please rename it to make it obvious that this relates to DNI validation